packages/runtime/container-runtime/README.md

```diff
@@ -51,6 +51,52 @@ issue](https://github.com/microsoft/FluidFramework/wiki/Submitting-Bugs-and-Feat
 
 Thank you!
 
+## Data Virtualization For DataStores (Work in Progress)
+
+It's a capability to exclude some content from initial snapshot (used when loading container) and thus improve boot
+performance of a container. Excluded content could be loaded at a later time when it's required.
+
+### Motivation for Data Virtualization
+
+This section talks about how the system used to work before Data virtualization. Fluid (through its APIs) does expose
+some levels of data virtualization. But under covers, there is almost none. The whole content of the file is downloaded
+in one go. Due to limitation of data virtualization, FF holds ALL blobs in snapshot as those might be required in the
```

Comment:
>blobs [](http://example.com/codeflow?start=66&length=5)

blobs and datastores?

Reply:
Datastore Context object is still there but we don't want to talk about that here in readme as that is internal concept.
Talking about blobs to user is fine as that is the main content of the snapshot which takes space.

On that note, I am reconsidering whether we want to tell what happened before as that will become obsolete once we have DV as Vlad mentioned. It is also making it a bit verbose.

Reply:
Okay Thanks.
For the 2nd half, you can say "Without DV" this is what happens and "With DV" this is how it will happen.